app/assets/javascripts/discourse/app/components/emoji-uploader.hbs

```diff
@@ -37,7 +37,7 @@
         disabled={{this.uploading}}
         type="file"
         multiple="true"
-        accept=".png,.gif"
+        accept="image/*, .heif, .heic"
```

Comment:
I don't think we want any other image formats here other than .png or .gif though, custom emojis are quite specific and we don't want larger formats like tiff, heic etc. I think the correct fix would just be to enforce only .png or .gif server-side for custom emojis